content/articles/blog/drizzle.md

```diff
@@ -0,0 +1,227 @@
+worse, i.e. introduced a performance regression, that would be treated
+the same as a functional regression. So you not only would be unable
+to land a patch that actually broke functionality or made the database
+server eat data; you would also be unable to land a patch that made
+the server slower.
+
+The Drizzle team is also where, to the best of my knowledge, a coinage
+for this kind of approach originated: "gated commits", or "gating" in
+general.
+
+## How is this relevant?
+
+A substantial fraction on the Drizzle core team — which had moved to
+Rackspace in 2010 — was instrumental in launching another project that
+came out of that company (and NASA) that same year:
+[OpenStack](https://openstack.org). And OpenStack took the gating
+approach from its humble beginnings with Drizzle to an absolutely
+massive scale in its hype years (2011 – 2015 or thereabouts) — so much
+so that it established a new default in collaborative software
+projects. Many other projects that launched in that timeframe
+(including [Kubernetes](https://kubernetes.io) and
+[Terraform](https://www.terraform.io)) adopted this approach as well.
+
+Today, having automated CI testing on every submitted patch is
+considered par for the course in a collaborative software
+project. [GitLab CI](https://docs.gitlab.com/ee/ci/) and [GitHub
+Actions
+workflows](https://docs.github.com/en/actions/automating-builds-and-tests/about-continuous-integration)
+have made these much more accessible than they used to be with Hudson
+and Jenkins. It's also exceedingly common to do detailed collaborative
+reviews in a public forum before merging — GitHub's PR review workflow
+is ever more closely approaching the Gerrit review workflow that
+OpenStack uses. GitHub's [auto-merge
+functionality](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)
+(which lands patches automatically once they have passed both
+automated unit tests and human review) is more or less a direct copy
+of the automated merge found in OpenStack's [development
+workflow](https://docs.opendev.org/opendev/infra-manual/latest/developers.html#development-workflow),
+which itself can be directly traced back to Drizzle's review process.
+
+And all these things are found in open source software projects across
+all sorts of communities. Kubernetes, Terraform,
+[Django](https://www.djangoproject.com/),
+[CPython](https://github.com/python/cpython), [Open
+edX](https://openedx.org) — you name it, it probably uses an approach
+first pioneered in Drizzle.
+
+And that's the real lasting legacy of a project that few people even
+remember by name.
+
+## Who do we owe this to?
+
+I know some of the Drizzle developers personally, though certainly not
+all. What follows is an incomplete list of people you can buy a meal
+or a drink if you run into them, and you like the way you
+collaboratively develop software today:
+
```

Comment:
I feel like an absolute pillock for not including Monty in this list. I even tagged him in the tweet! 🤦‍♂️

The others I didn't know about. Thank you! Patch updated.

Reply:
https://github.com/stewartsmith/drizzle/graphs/contributors has a fair amount of commits in 2009 from Pádraig O'Sullivan; probably a good idea to include him?

And I included Eric Day because he was mentioned [here](https://sheeri.org/video-eric-day-and-patrick-galbraith-speak-about-drizzle-and-gearman/) but I don't know how substantial his early contributions were.

Reply:
Eric worked a lot on early connectors, it's fair to include.

Reply:
and yes, also to Pádraig

Reply:
Done. Thanks!